ql/src/java/org/apache/hadoop/hive/ql/exec/repl/AtlasDumpTask.java

```diff
@@ -73,7 +74,12 @@ public int execute() {
       String entityGuid = checkHiveEntityGuid(atlasRequestBuilder, atlasReplInfo.getSrcCluster(),
               atlasReplInfo.getSrcDB());
       long currentModifiedTime = getCurrentTimestamp(atlasReplInfo, entityGuid);
-      dumpAtlasMetaData(atlasRequestBuilder, atlasReplInfo);
+      AtlasDumpLogger replLogger = new AtlasDumpLogger(atlasReplInfo.getSrcDB(),
+              atlasReplInfo.getStagingDir().toString());
+      replLogger.startLog();
+      long numBytesWritten = dumpAtlasMetaData(atlasRequestBuilder, atlasReplInfo);
+      LOG.debug("Finished dumping atlas metadata, total:{} bytes written", numBytesWritten);
+      replLogger.endLog(0L);
```

Comment:
Do you want to send 0 or send the bytes written? Not sure how useful that is though

Reply:
Yes, need it public for test.

Reply:
We don't need this return value. This is just a workaround for the mockito bug where it is calling a method of a mocked object. Like the one we had seen during atlas repl initial patch.